<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Животное:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Птица:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Пингвин:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_animal_create">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать животное</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_animal_rename">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Переименовать животное</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_animal_number_change">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить количество жиотных</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_animal_rb_status_change">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить статус животного в КК</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_animal_live">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Животное, живи!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_animal_showinfo">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать инфо о Животном</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_bird_rb_status_change">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>190</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить статус в КК</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_bird_live">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Птица, живи!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_bird_create">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать птицу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_bird_number_change">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить количество птиц</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_bird_rename">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Переименовать птицу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_bird_showinfo">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>230</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать инфо о Птице</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_bird_fly">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Птица, лети!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_bird_predator_status_change">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>190</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить корм (хищник: да/нет)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_name">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Животное</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_pinguin_create">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать пингвина</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_pinguin_number_change">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить количество пингвинов</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_red_book">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>нет</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_number">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Хищник (да/нет):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_pinguin_rb_status_change">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить статус в КК</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_pinguin_fly">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>380</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Пингвин, лети!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>В Красной книге (да/нет):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_pinguin_showinfo">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>380</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать инфо о Пингвине</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_pinguin_predator_status_change">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить корм (хищник: да/нет)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_predator">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Название:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_pinguin_rename">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Переименовать пингвина</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_pinguin_live">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>380</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Пингвин, живи!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Место обитания:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_living_place">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="push_pinguin_living_place_change">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>340</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить место обитания</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
